siliconcompiler/toolscripts/ubuntu20/install-klayout.sh

```
@@ -5,16 +5,17 @@ set -ex
 # Get directory of script
 src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)/..
 
+# Install prerequisites only when they are missing
+. "${src_path}/_prereqs.sh"
+
 USE_SUDO_INSTALL="${USE_SUDO_INSTALL:-yes}"
 if [ "${USE_SUDO_INSTALL:-yes}" = "yes" ]; then
     SUDO_INSTALL=sudo
 else
     SUDO_INSTALL=""
```

Shellcheck (0.11.0)

[warning] SUDO_INSTALL appears unused. Verify use (or export if used externally).

(SC2034)